msresist/figures/figureM2.py

```diff
@@ -265,43 +269,54 @@ def ErrorAcrossNumberOfClusters(distance_method):
 
 def ErrorAcrossWeights(distance_method):
     """Calculate missingness error across different number of clusters."""
+    models = []
     if distance_method == "PAM250":
-        with open('msresist/data/pickled_models/CPTACmodel_PAM250_filteredTMT', 'rb') as m:
-            model = pickle.load(m)[0]
-        weights = [0, 1, 3, 9, 27]
+        with open('msresist/data/pickled_models/CPTACmodel_PAM250_CL21_W0_TMT2', 'rb') as m:
+            models.append(pickle.load(m)[0])
+        with open('msresist/data/pickled_models/CPTACmodel_PAM250_CL21_W3_TMT2', 'rb') as m:
```

Comment:
@mcreixell why do you need to unpickle all of these at once? If you unpickle, analyze the model, and then overwrite the first model when you unpickle the next, you won't have this problem.

Reply:
True, the only thing is that it seemed like it was breaking while unpickling the first model but it's definitely worth a shot. I will try now.

Reply:
That would indicate I'm wrong—how did you determine that?

Reply:
By tracing back the error. It pointed at the line of the first model to be loaded.

Reply:
I just tried re-running and the server froze and asked me to reload instead of breaking due to an error. I think it's worth trying what you're suggesting. Should be quick.